lib/Tab.js

```diff
@@ -299,11 +299,12 @@ const styles = StyleSheet.create({
     flex: 1,
     alignItems: 'center',
     paddingTop: 8,
-    paddingBottom: 10,
+    paddingBottom: (Platform.OS !== 'web' ? 10 : 0),
     paddingLeft: 12,
     paddingRight: 12,
     backgroundColor: 'transparent',
     position: "relative",
+    ...(Platform.OS !== 'web' ? {} : { transition: '0.5s all' }),
```

Comment:
Just a comment, no change request:

Using `transition` to animate `flex` isn't really performant. https://csstriggers.com/flex-grow has a good explanation why. On a mobile browser with a mediocre phone, this could result in a laggy animation. But for now it's a very simple addition and I can't think of a better way from the top of my head.

Reply:
A method to make very smooth animations on web is FLIP. Here's the original article by Paul Lewis: https://aerotwist.com/blog/flip-your-animations/

I think for the scope of this PR, using `transition` should be enough. Maybe anything else is a bit too crazy.